angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.html

```diff
@@ -31,6 +31,11 @@ <h2 class="border-0 mb-0">Edit Collection</h2>
           </div>
         </div>
         <div class="col-6 col-lg-4 col-xl-3">
+          <div class="d-flex justify-content-end text-left">
+            <span *ngIf="isEditing" class="grey-subtext d-block">
+              Cancel will cause you to lose any records or links added to this collection - if you do not want to lose the added items click update
```

Comment:
Not sure if we want a period on this or not.

Reply:
```Cancel will cause you to lose any records or links added to this collection.  If you do not want to lose the added items, click update.```

Reply:
👍